libraries/botbuilder/src/botFrameworkHttpClient.ts

```diff
@@ -116,7 +123,7 @@ export class BotFrameworkHttpClient extends BotFrameworkClient {
         }
     }
 
-    protected async buildCredentials(appId: string, oAuthScope?: string): Promise<MicrosoftAppCredentials> {
+    protected async buildCredentials(appId: string, oAuthScope?: string): Promise<AppCredentials> {
```

Comment:
Just confirming that 4.7 doesn't have a buildCredentials method with MicrosoftAPpCredentials as a return type. Protected is still public for folks who extend the API.

Reply:
Technically this is loosening the the API as MicrosoftAppCredentials subclasses AppCredentials... 

But this technically is a breaking change if users now have to cast the return value from their code... Thoughts?

Reply:
Additional caveat, this makes the signature the same as C#: [link](https://github.com/microsoft/botbuilder-dotnet/blob/2ebf6d118ceebece83ff7b5b38edbc2d8a253679/libraries/integration/Microsoft.Bot.Builder.Integration.AspNet.Core/BotFrameworkHttpClient.cs#L146)